opentelemetry-sdk/tests/context/propagation/test_b3_format.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import opentelemetry.sdk.context.propagation.b3_format as b3_format
+import opentelemetry.sdk.trace as trace
+
+FORMAT = b3_format.B3Format()
+
+
+def _get_from_dict(carrier: dict, key: str) -> str:
```

Comment:
Can you use [`getitem`/`setitem`](https://docs.python.org/3/library/operator.html) here?

Reply:
It does look like get / __setitem__ works. I'll use those, thanks!